trillian/ctfe/configpb/config.proto

```diff
@@ -43,12 +43,19 @@ message LogConfigSet {
 
 // LogConfig describes the configuration options for a log instance.
 message LogConfig {
+  // The ID of a Trillian tree that stores the log data. The tree type is LOG
+  // for regular CT logs, and PREORDERED_LOG for mirror logs.
   int64 log_id = 1;
+  // TODO(pavelkalinnikov): Comment this.
   string prefix = 2;
+  // Paths to the files containing root certificates that are acceptable to the
+  // log. The certs are served through get-roots endpoint. Optional in mirrors.
```

Comment:
Optional or not required?

Reply:
Both!

What's the difference? Not required == must be null?

Reply:
Or rather "not required" == "is not used, so we don't care if it's null or not", whereas "optional" == "we will use it if you provide it"?

As it stands, I'd say it's "optional", because if the files are specified, CTFE will serve them regardless of the log type.